bin/review-changes

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# [review] [changes] in repositories over a given time period.
+#
+# Example:
+#  review-changes 1.week
+
+set -euo pipefail # exit on any error, don't allow undefined variables, pipes don't swallow errors
+
+time_period=$1
+
+cd ~/code
+
+for dir in $(my-repos) ; do
+  cd "$dir"
+  blue "$dir"
+
+  DELTA_PAGER="cat" git diff "HEAD@{$time_period.ago}" -- . \
+    ':(exclude)Gemfile.lock' \
+    ':(exclude)yarn.lock' \
+    ':(exclude)pnpm-lock.yaml' \
+    ':(exclude)package-lock.json' \
+    ':(exclude)**/Gemfile.lock' \
```

Comment:
We want to ignore lock files in subdirectories, as well, for monorepos. But it seems that this glob doesn't exclude top-level package lock files, which is why we also list each of these above without a `**/` prefix.